src/commons/sagas/WorkspaceSaga.ts

```diff
@@ -1256,6 +1257,14 @@ export function* evalCode(
 
   yield* dumpDisplayBuffer(workspaceLocation, isStoriesBlock, storyEnv);
 
+  // Change token count if its assessment and EVAL_EDITOR
+  if (actionType === EVAL_EDITOR && workspaceLocation === 'assessment') {
+    let tokenCounter = 0;
+    const tokens = [...tokenizer(entrypointCode, ACORN_PARSE_OPTIONS)];
+    tokenCounter = tokens.length;
```

Comment:
Can we just

```ts
const tokenCount = tokens.length;
```

instead of declaring it at the top?

Reply:
Yes, I'll make sure to edit that.